spec/features/production/deployment_spec.rb

```diff
@@ -1,8 +1,8 @@
 require "spec_helper"
 
-RSpec.describe "suspenders:production:deployment", type: :generator do
+RSpec.describe "suspenders:production:deployment", type: :feature do
   it "generates the configuration for a production deployment" do
-    rm "bin/deploy"
+    rm_file "bin/deploy"
```

Comment:
I'm taking the opportunity to make these method names more consistent. They should all be suffixed with `_file`.